src/Mapping/ExtensionMetadataFactory.php

```diff
@@ -137,6 +138,12 @@ public static function getCacheId($className, $extensionNamespace)
      */
     protected function getDriver($omDriver)
     {
+        if ($omDriver instanceof DoctrineBundleMappingDriver) {
+            $omDriver = (new \ReflectionClass($omDriver))
+                ->getProperty('driver')
+                ->getValue($omDriver);
```

Comment:
Getting the following error:

> In FileLoader.php line 173:
>
>  Cannot access non-public property Doctrine\Bundle\DoctrineBundle\Mapping\MappingDriver::$driver in /sulu-cms.localhost/vendor/sulu/sulu/src/Sulu/Bun
  dle/ContactBundle/Resources/config/routing_api.yml (which is being imported from "/Users/sulu-cms.localhost/config/routes/sulu_admin.yaml"). Make sure the
>  re is a loader supporting the "rest" type.
>
>
> In ExtensionMetadataFactory.php line 145:
>
>  Cannot access non-public property Doctrine\Bundle\DoctrineBundle\Mapping\MappingDriver::$driver

I think here is the `setAccessible` call missing:

```suggestion
            $propertyReflection = (new \ReflectionClass($omDriver))
                ->getProperty('driver');
            $propertyReflection->setAccessible(true);
            $omDriver = $propertyReflection->getValue($omDriver);
```